raspbian/applications/camera.md

```diff
@@ -448,7 +448,9 @@ Outputs debugging/information messages during the program run.
 	--timeout,	-t		Time before the camera takes picture and shuts down
 ```
 
-The program will run for this length of time, then take the capture (if output is specified). If not specified, this is set to 5 seconds.
+The program will run for this length of time, in milliseconds, then take the capture, which will be saved if an output is specified. If a timeout is not specified, then it is set to 5 seconds (-t 5000). Note that low values (less than 500ms, although it can depend on other settings) may not give enough time for the camera to start up and provide enough frames for the automatic algorithms like AWB and AGC to provide accurate results.
```

Comment:
Feels like maybe there's too many commas in the first sentence? Only a minor niggle though.

"may not give enough time for the camera to start up and provide enough frames for the automatic algorithms like AWB and AGC to provide accurate results" - is that exactly what the `--stats` option is for? (if so, maybe it's worth mentioning that? Or are they slightly different things?)

Reply:
Different things.

Reply:
Have rewritten first sentence.